docs/components/multiselect/index.md

```diff
@@ -0,0 +1,308 @@
+
+```hbs
+<Form::Controls::Multiselect
+  @onFilter={{this.handleFilter}}
+  @onChange={{this.handleChange}}
+  @options={{this.options}}
+  @optionKey='label'
+  @selected={{this.selected}}
+  as |multiselect|
+>
+  <multiselect.Option>
+    {{multiselect.option}}
+  </multiselect.Option>
+</Form::Controls::Multiselect>
+```
+
+```js
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class extends Component {
+  @tracked selected;
+
+  options = [
+    {
+      label: 'Blue',
+      value: 'blue',
+    },
+    {
+      label: 'Green',
+      value: 'green',
+    },
+    {
+      label: 'Yellow',
+      value: 'yellow',
+    },
+    {
+      label: 'Orange',
+      value: 'orange',
+    },
+    {
+      label: 'Red',
+      value: 'red',
+    },
+    {
+      label: 'Purple',
+      value: 'purple',
+    },
+    {
+      label: 'Teal',
+      value: 'teal',
+    },
+  ];
+
+  @action
+  handleChange(option) {
+    this.selected = option;
+  }
+
+  @action
+  handleFilter(value) {
+    return this.options.filter((option) => option.label === value);
+  }
+}
+```
+
+## Disabled State
+
+Set the `@isDisabled` argument to disable the input.
+
+```hbs
+<Form::Controls::Multiselect @isDisabled={{true}} />
+```
+
+## Read Only State
+
+Set the `@isReadOnly` argument to put the input in the read only state.
+
+```hbs
+<Form::Controls::Multiselect @isReadOnly={{true}} />
+```
+
+## Error State
+
+Set the `@hasError` argument to apply an error box shadow to the `<input>`.
+
+```hbs
+<Form::Controls::Multiselect @hasError={{true}} />
+```
+
+## Test Selectors
+
+### Container
+
+`data-multiselect-container` is provided to target the wrapping div element for the entire component.
+
+```js
+assert.dom('[data-multiselect-container]').hasClass('bg-overlay');
+```
+
+### Selected Chips
+
+Target selected chips via `data-multiselect-selected-option`.
+
+```js
+// Query how many selected chips there are
+assert.dom('[data-multiselect-selected-option]').exists({ count: 2 });
+```
+
+### Select Chip Remove Button
+
+Target a selected chip's remove button via `data-multiselect-remove-option`.
+
+```js
+// Query how many remove buttons there are
+assert.dom('[data-multiselect-remove-option]').exists({ count: 3 });
+
+// Click a particular remove button
+let removeButtons = document.querySelectorAll(
+  '[data-multiselect-remove-option]'
+);
+await click(removeButtons[1]);
+```
```

Comment:
I think ultimately we probably want to add a fractal-page-object (downstream PR (?)) for folks to interact with the multiselect rather than relying on these data attributes.  This is probably fine, imo, for a first pass unless folks have strong opinions.